platform_api/orchestrator/kube_orchestrator.py

```diff
@@ -875,3 +862,7 @@ async def delete_all_job_resources(self, job_id: str) -> None:
         await self._client.delete_all_ingresses(labels=labels)
         await self._client.delete_all_services(labels=labels)
         await self._client.delete_all_network_policies(labels=labels)
+
+    async def preempt_idle_jobs(self, jobs_to_schedule: list[Job]) -> None:
```

Comment:
Currently not used anywhere. Requires K8s upgrade to at least 1.19 of all clusters